syntax = "proto3";

message MySegmentArgs {
  float sigma1 = 3;
  float sigma2 = 4;
  string model_path = 5;
}

message ProtoImage {
  int32 w = 1;
  int32 h = 2;
  bytes image_data = 3;
}